frontend/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "frontend",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://jibranabsarulislam.github.io/Fornear/",
+  "homepage": "https://jibranabsarulislam.github.io/",
```

Comment:
this works. i will say that this homepage key came from  trying to use github pages during the hackathon, so we don't need it at all anymore. the entire line can be deleted

Reply:
okay will remove